lib/rom/sql/attribute.rb

```diff
@@ -235,6 +257,11 @@ def sql_expr
         @sql_expr ||= (meta[:sql_expr] || Sequel[to_sym])
       end
 
+      # @api private
+      def type_extensions
+        @type_extensions ||= TypeExtensions[unwrap_type(type)] || EMPTY_HASH
+      end
```

Comment:
Right, this makes sense, I just wanted to show that we need to unwrap it _somewhere_ anyway. I'll move unwrapping to `TypeExtenions.[]`